### ISSUE TITLE
Feedback on `pre_release()`

### ISSUE THREAD

**Issue by pat-s** [krlmlr/fledge#35]
## General

- Should win-builder run be automatically triggered? (if possible). Could be a logical arg in `pre_release()`
- > After CRAN release, call `fledge::tag_version()` and `fledge::bump_version()` to re-enter development mode`
 
  -> Is this still valid or accounted for by `release()` and `post_release()`?

## cran-comments.md

```
mlr3verse 0.1.2

## Cran Repository Policy

- [x] Reviewed CRP last edited 2020-05-21.

## R CMD check results

- [x] Checked locally, R 4.0.1
- [x] Checked on CI system, R 4.0.1
- [ ] Checked on win-builder, R devel

OK

- [ ] Explain R CMD check failures or remove this line.

## CRAN failures

- [x] Checked on 2020-06-15, errors found: https://cran.r-project.org/web/checks/check_results_mlr3verse.html
- [ ] NOTE: r-devel-linux-x86_64-debian-clang, r-devel-linux-x86_64-debian-gcc, r-devel-linux-x86_64-fedora-clang, r-devel-linux-x86_64-fedora-gcc, r-devel-windows-ix86+x86_64, r-patched-linux-x86_64, r-patched-solaris-x86, r-release-linux-x86_64, r-release-osx-x86_64, r-release-windows-ix86+x86_64, r-oldrel-osx-x86_64, r-oldrel-windows-ix86+x86_64

Check results at: https://cran.r-project.org/web/checks/check_results_mlr3verse.html
```

- What does the "OK" mean here?

- It says "errors found:" whereas in fact only notes were found. Also I am a bit confused why there is a checkbox. Should it be only a bullet point?

**Comment by krlmlr** [comment]
1. On RStudio, the command to be run is sent to the console. Does it work for you?
2. Can we safely refer to `post_release()` in all cases?
3. OK means that local, CI and win-builder checks were run and returned OK (otherwise we would not release or would have to explain). Feel free to adapt the template.
4. Do we need more logic for the template?

**Comment by pat-s** [comment]
> On RStudio, the command to be run is sent to the console. Does it work for you?

Nope. Looked at the code and not sure why it was not triggered. No exp yet with `callr::r_bg()` though.

> Can we safely refer to post_release() in all cases?

Haven't arrived there yet, feedback later.

> OK means that local, CI and win-builder checks were run and returned OK (otherwise we would not release or would have to explain). Feel free to adapt the template.

Ah ok. If they are triggered and then to be ticked off by the user once they finished, this works and makes sense.

Would it make sense to replace "OK" with "Tick off and remove message if win-builder and CI build passed"?

---

> Checked on 2020-06-15, errors found: https://cran.r-project.org/web/checks/check_results_mlr3verse.html

"errors found" is confusing if there are only notes for the CRAN checks.
I think `get_cransplainer()` should be more concrete here and list the most severe level found, i.e. error > warning > note

In addition, I'd change "CRAN failures" to "Current CRAN check results".

Also, just use bullet points here instead of checkboxes?

**Comment by krlmlr** [comment]
Need to run `devtools::document()` and render `README.Rmd` if it exists.

**Comment by pat-s** [comment]
> checks <- callr::r_bg(function() { devtools::check_win_devel(quiet = TRUE); rhub::check_for_cran() })

Sometimes I want to sent the winbuilder results to a different email. Even if email-forwarding-automation is set up for CRAN release mails of the maintainer, it would be great if we could forward the `email` arg of `devtools::check_win_devel()` and `rhub::check_for_cran()` from within `pre_release()`.

**Comment by maelle** [comment]
related https://github.com/cynkra/fledge/issues/689#issue-1785551176
